### ISSUE TITLE
Disabling audio preloading

### ISSUE THREAD

**Issue by lestephane** [mattstratton/castanet#241]
It seems that the mp3s are being preloaded, so when I go to the main home page which lists a bunch of mp3s, the browser starts downloading them all, messing with my download stats. Can this behaviour be disabled? There's some magic happening with audiojs that I don't understand. Grepping for audio elements does not help:

```text
themes/castanet-1.1.24$ grep -rli "audio" | grep preload
themes/castanet-1.1.24$ grep -rli "audio" | grep auto
themes/castanet-1.1.24$
```
(I actually want to get rid of audiojs altogether, as I don't want the swf that it brings with it, and everyone supports HTML5 nowadays, but that's another issue)

**Comment by mattstratton** [comment]
This is a good point (and audiojs is kind of deprecated IIRC, so I've been wanting to replace it with something else anyway). I'll look into this!

**Comment by mattstratton** [comment]
If there's a different library you suggest, I'd love the feedback! (I am looking myself, but happy to take any suggestions as well to help speed this along)

**Comment by mattstratton** [comment]
I'm thinking that https://github.com/mediaelement/mediaelement is probably a pretty good choice - it also allows to disable preloading, so that is a plus. I'll experiment it with it today or tomorrow.

**Comment by mattstratton** [comment]
Note to self - there has been a request to add playback speed controls, so remember this, Matt :)

https://github.com/mediaelement/mediaelement-plugins/blob/master/docs/speed.md

**Comment by mattstratton** [comment]
@lestephane this was fixed in 1.2.0.

Of course, in the process I saw how to disable it on the old player, but I wanted to update the player used anyway :)

**Comment by lestephane** [comment]
Thanks! It may take a while for me to try it, i just realized i'm too far back and need to do some migration from castanet-0.14.1 to castanet-1.1.24.
